src/mpi_native/mpi_native.cpp

```diff
@@ -47,28 +65,777 @@ int MPI_Init(int* argc, char*** argv)
 int MPI_Comm_rank(MPI_Comm comm, int* rank)
 {
     auto logger = faabric::util::getLogger();
-    logger->debug("MPI_Comm_rank");
+    logger->debug("S - MPI_Comm_rank");
```

Comment:
This `S - ` syntax is a hangover from Faasm's host interface implementation and doesn't really mean anything any more. What value does it add here? If we want to make log parsing easier we could come up with a better scheme (e.g. using different loggers for different system components, or having a `[MPI]` prefix for MPI-related logs.

Reply:
This will be addressed in #55